modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -2,19 +2,66 @@
 
 This kernel adds the friction term to the porous media Navier Stokes momentum
 equations. This kernel must be used with the canonical PINSFV variable set,
-e.g. pressure and superficial velocity.  A variety of models are supported for
-the friction force:
+e.g. pressure and superficial velocity. This kernel supports Darcy and
+Forchheimer friction models:
 
 Darcy drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_d
+F_i = - f_i \frac{\rho v_i}{\epsilon}
```

Comment:
indeed should be epsilon F, it s a matter of definition but this is more consistent. 
what we add to residual is epsilon F or for Forchheimer
f rho v v / Dh / eps = eps * (f rho v v / Dh / eps^2) = eps * F

this is missing some constant btw
```suggestion
F_i = - f_i \frac{\rho v_{D,i}}{\epsilon}
```

Reply:
I ll make the replacing